recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -52,6 +56,23 @@ PACKAGECONFIG[load-tests] = "-DBUILD_LOAD_TESTS=ON,-DBUILD_LOAD_TESTS=OFF,"
 PACKAGECONFIG[serialcan] = ",,,slcand-start"
 PACKAGECONFIG[ubootenv] = ",,,u-boot-fw-utils aktualizr-uboot-env-rollback"
 
+do_compile_ptest() {
+	cmake_runcmake_build --target build_tests
+}
+
+do_install_ptest() {
+	# copy the complete source directory (contains build)
+	cp -r ${B}/ ${D}/${PTEST_PATH}/build
+	cp -r ${S}/ ${D}/${PTEST_PATH}/src
+
+	# remove bogus elf file
+	rm ${D}/${PTEST_PATH}/src/partial/extern/RIOT/cpu/esp32/bin/bootloader.elf
+
+	# fix the absolute paths
+	find ${D}/${PTEST_PATH}/build -name "CMakeFiles" | xargs rm -rf
+	find ${D}/${PTEST_PATH}/build -name "*.cmake" -or -name "DartConfiguration.tcl" -or -name "run-valgrind" | xargs sed -e "s|${S}|${PTEST_PATH}/src|g" -e "s|${B}|${PTEST_PATH}/build|g" -e "s|\"--gtest_output[^\"]*\"||g" -i
+}
+
```

Comment:
Nice. Very minor complaint: indentation differs from the rest of this file.

Reply:
sorry, fixed :)